### ISSUE TITLE
Support Links in description of Use Case / Scenario / Step

### ISSUE THREAD

**Issue by adiherzog** [scenarioo/scenarioo#391]
It should be possible to add a link to a description of a Use Case / Scenario / Step.

There are two variants:
* Allow and parse HTML so that the generator can add <a href...></>
* Auto-Detect URLs and make them into links (whenever a String starts with http:// or https:// it can be turned into a link that opens in a new window).

I prefer the second variant. What do you think? @bruderol @danielsuter @Fredericson

**Comment by bruderol** [comment]
I agree. I also prefer the second variant.

Additionaly (maybe as a separate story?) I think it would be great if markdown syntax is allowed in descriptions. What do you think about this?
This would have the advantage that one could give a link a nice text or use bullets for longer texts etc.

By the way ... don't we allready allow html tags??
At least in one project we even included images somehow in some metadata descriptions and it displayed correctly (maybe only working there??).

We should be consistens about this in all places where we display content.

**Comment by adiherzog** [comment]
Yeah, in the metadata it seems that HTML already works. But in the descriptions it does not. So we should probably do both proposed variants.

**Comment by bruderol** [comment]
would be cool, if we could put that in the next release 3.0 - I put it to the release 3.0 milestone, we can still reprioritize it. But it is still marked as "Backlog", because not yet ready to implement.

**Comment by bruderol** [comment]
best would be if mardown syntax could be used as well.
